ui/app/templates/vault/cluster/license.hbs

```diff
@@ -0,0 +1,9 @@
+{{license-info
```

Comment:
We can use the new angle bracket syntax for components now - we should write up some standards around that and expectations of use, etc. (no need to change this now, just a note to chat with you later about what standards we want)